native/metrodroid/metrodroid/Assets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can the assets in this directory be automatically generated?  That way we don't have to run Xcode every time there is a new card.

Reply:
The ones from png are just a symlink and a Contents.json that differs only in file name. The real trouble are vector one. We need to store xml (Android) and png (iOS) for every vector image.I'm unsure what the right approach should be.

Reply:
Fair enough -- lets fix this later.